lib/swaps/Swaps.ts

```diff
@@ -342,13 +394,37 @@ class Swaps extends EventEmitter {
     } else {
       // If we are here we are the taker
       this.logger.debug('Executing taker code');
+
+      // Verify that taker got right amount.
```

Comment:
We have `Verify that taker got right amount.` and `verify taker's takerCltvDelta` twice in this file. The logic of those things should be put into a function to avoid redundancy.

Reply:
OK boss.